.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,45 @@
+name: publish-prerelease
+on:
+  workflow_dispatch:
+    inputs:
+      pre-release-tag:
+        description: 'The pre-release tag use in the version'
+        required: false
+        default: 'pre'
+      dist-tag:
+        description: 'The dist-tag use'
+        required: false
+        default: 'next'
+      dependencies-to-update:
+        description: 'csv of dependencies to update with the dist-tag'
+        required: false
+        default: '@adobe/aio-lib-runtime','@adobe/aio-lib-web'
```

Comment:
the value should be a one string csv: '@adobe/aio-lib-runtime,@adobe/aio-lib-web'

Reply:
Okay, fixed that